instrumentation/volley/library/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+import java.time.Duration
+
+plugins {
+    id("com.android.library")
+    id("otel.android-library-conventions")
+    id("otel.errorprone-conventions")
+}
+
+android {
+    namespace = "io.opentelemetry.android.volley"
+
+    compileSdk = 34
+    buildToolsVersion = "34.0.0"
+
+    defaultConfig {
+        minSdk = 21
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+    compileOptions {
```

Comment:
All this `compileOptions` block is already defined in the conventions so it's not needed to define it here too.